.github/workflows/rust.yml

```diff
@@ -27,7 +28,7 @@ jobs:
       - name: Running tests on ${{ matrix.toolchain }}
```

Comment:
You can do this to avoid running on macos:

```
if: ${{ matrix.os != 'macOS-latest' }}
```

Also, I've seen it spelled `macos-latest` before, but maybe that doesn't matter.

Reply:
How do I prevent specifically rustc 1.41 from running on macos?

Reply:
Ah, sorry, I didn't realize that the combination of  macos and 1.41. I didn't test this, but it should be something like:

```
if: ${{ !(matrix.os == 'macOS-latest' && matrix.toolchain == `1.4.1`) }}
```

Reply:
It's a huge PITA to test github actions :) bash is easier.